README.md

```diff
@@ -178,7 +178,12 @@ make docker-secrets
 # Build base images
 make build-common-base
 
-# And open up the cluster's gateway with
+# If you are running the bentoV2 with the use of an external identity provider, i.e setting BENTOV2_USE_EXTERNAL_IDP=1, run both
+make run-gateway
+# and
+make auth-setup
+
+# Otherwise, only open the cluster's gateway with
```

Comment:
It seems contradictory with the next line. Should it be `make run-gateway` instead?

Reply:
I see the confusion.
This is correct, in that `auth-setup` in the second case still does perform important configuration, despite the command name, that need be done before spinning up the gateway container (see how the nginx.conf template differs in both cases). The main difference is that when running with a local auth provider like KC, it requires the gateway in its base default setting to be up and running so the one of the automatic scripts can wire them together.

Reply:
Thanks for the clarification!